modules/cache-common/src/main/java/org/opensearch/cache/common/tier/TieredSpilloverCache.java

```diff
@@ -47,6 +49,9 @@
 @ExperimentalApi
 public class TieredSpilloverCache<K, V> implements ICache<K, V> {
 
+    // Used to avoid caching stale entries in lower tiers.
+    private static final List<RemovalReason> REMOVAL_REASONS_FOR_EVICTION = List.of(RemovalReason.EVICTED, RemovalReason.CAPACITY);
```

Comment:
Can you come up with a better name for this? It doesn't communicate that these are the eviction reasons that will trigger spillover to the next tier.

Maybe `SPILLOVER_REMOVAL_REASONS`?